app/assets/main/react_components/Button.jsx

```diff
@@ -27,8 +27,8 @@ export default class Button extends Component {
         )
       }
 
-      return(
-        <button 
+      return (
+        <button
```

Comment:
Avoid adding extra space, check git Working Tree to see these changes and go through and revert accidental/unnecessary spaces and indents.